src/condor_utils/user_job_policy.cpp

```diff
@@ -467,7 +468,7 @@ UserPolicy::AnalyzePolicy(ClassAd & ad, int mode)
 				if( time(NULL) - birthday >= allowedJobDuration ) {
 					m_fire_expr = ATTR_JOB_ALLOWED_JOB_DURATION;
 					m_fire_source = FS_JobDuration;
-					formatstr(m_fire_reason, "The job exceeded allowed job duration of %d", allowedJobDuration);
+					formatstr(m_fire_reason, "The job exceeded allowed job duration:%s", format_time(allowedJobDuration));
```

Comment:
" of %s" would be better than "duration:%s" here, at the very least we need some whitespace before %s